.changeset/tidy-cooks-mix.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-azure-sites-backend': minor
+'@backstage/plugin-azure-sites-common': minor
+'@backstage/plugin-azure-sites': minor
+---
+
+Implemented Permission framework to protect start and stop `Azure sites`.
```

Comment:
This changeset is marked as `minor` which effectively means breaking changes.

I think that there are some breaking changes in these changes, with now requiring things like the `catalogApi` and other things that integrators might have to change.

Can you highlight some code examples where there could be breaking changes and their remediations?

If there's no breaking changes then I think that these changesets should be `patch` instead?

Reply:
Yes, It is breaking change. I will added the details along with code example.